chaincode/marbles_transfer/marbles_transfer.go

```diff
@@ -33,6 +33,7 @@ import (
 const (
 	typeMarbleForSale = "S"
 	typeMarbleBid     = "B"
+	typeMarbleReceipt = "R"
```

Comment:
Let's use a different prefix for sales receipt versus buy receipt.

Reply:
How about `BR` and `SR`?